### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by kamfur00** [NightscoutFoundation/xDrip#2770] [Q&A]
Please, help. I've been trying to connect MiBand 6 with xDrip+, Bubble mini and Samsung A31 for a week. I use WatchDrip+. Everything starts to work, but as soon as MiBand 6 loses Bluetooth connection with the phone (I’ll leave the phone for 20 minutes), synchronization starts, it reaches 70%, and the firmware in MiBand flies to stock, and the phone stops seeing MiBand, and most importantly, it doesn’t see Bubble Mini. At the same time, Bubble Mini connects at best after rebooting the phone. Help me please. Thank you

**Comment by Navid200** [comment]
Please raise this with Artem,

If a change to xDrip is needed to address this, he will know how to do that.
If a change to WhatchDrip is needed to deal with it, he will know how to do it.

**Reply by kamfur00** [reply]
I wrote to him, but for some reason he does not answer ((((

**Reply by Navid200** [reply]
Have you posted your question here?  https://bigdigital.home.blog/2022/06/16/watchdrip-a-new-application-for-xdrip-watch-integration/


If you scroll down, you see people asking for help and others providing support.
That seems a place you can start.
